tests/test_manifest_evolution.py

```diff
@@ -20,12 +20,12 @@ def examples():
     """Returns examples as tuples of component and expected output_manifest."""
     for directory in (f for f in examples_path.iterdir() if f.is_dir()):
         with open(directory / "component.yaml") as c, open(
-            directory / "output_manifest.json"
+            directory / "output_manifest.json",
         ) as o:
             yield yaml.safe_load(c), json.load(o)
 
 
-@pytest.mark.parametrize("component_spec, output_manifest", list(examples()))
```

Comment:
what ?

Reply:
This was fixed automatically. Is this not logical?

Reply:
I'm surprised it worked before the fix